mkt/search/views.py

```diff
@@ -101,6 +101,12 @@ def name_query(q):
     return query.Bool(should=should)
 
 
+def _get_regional_sort(region, field):
+    if region and not region.adolescent:
```

Comment:
Please add docstring explaining that mature regions get their own field to sort by.